Swiftfin tvOS/Components/LandscapePosterProgressBar.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2022 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct LandscapePosterProgressBar: View {
+
+    let title: String
+    let progress: CGFloat
+
+    var body: some View {
+        GeometryReader { _ in
```

Comment:
I don't think it's necessary.
Is there another reason?

Reply:
Good catch, I just copied over the iOS one and missed that.